products/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+class Product(models.Model):
+    name          = models.CharField(max_length=45)
+    price         = models.DecimalField(max_digits=18, decimal_places=2)
+    description   = models.TextField()
+    main_img_url  = models.URLField()
+    hover_img_url = models.URLField()
```

Comment:
```suggestion
    main_image_url  = models.URLField()
    hover_image_url = models.URLField()
```
변수명은 명확할수록 좋습니다. 줄이지 말고 사용해주세요!

Reply:
넵! 수정했습니다!